examples/rpc/rpc-server.cpp

```diff
@@ -0,0 +1,145 @@
+#ifdef GGML_USE_CUDA
+#include "ggml-cuda.h"
+#endif
+
+#ifdef GGML_USE_METAL
+#include "ggml-metal.h"
+#endif
+
+#include "ggml-rpc.h"
+#include <memory>
+#include <string>
+#ifndef _WIN32
+#  include <sys/socket.h>
+#  include <sys/types.h>
+#  include <arpa/inet.h>
+#  include <netinet/in.h>
+#  include <netinet/tcp.h>
+#  include <netdb.h>
+#  include <unistd.h>
+#endif
+#include <stdio.h>
+#include <stdlib.h>
+
+static ggml_backend_t create_backend() {
+    ggml_backend_t backend = NULL;
+#ifdef GGML_USE_CUDA
+    fprintf(stderr, "%s: using CUDA backend\n", __func__);
+    backend = ggml_backend_cuda_init(0); // init device 0
```

Comment:
It would be nice if there was a command line parameter to specify what device to use, although for now as a workaround using `CUDA_VISIBLE_DEVICES` would also work to select the device to use in multi-GPU systems.